Zend/zend_types.h

```diff
@@ -47,7 +47,6 @@
 # define ZEND_ENDIAN_LOHI_C_4(a, b, c, d)  a, b, c, d
 #endif
 
-typedef bool zend_bool;
 typedef unsigned char zend_uchar;
```

Comment:
should be probably deprecated as well

Reply:
I understand `zend_bool`, `zend_*inptr_t` were definitions from a time when C99 wasn't required; but `unsigned char` has always been available, everywhere. Therefore I figured this is not a compatibility definition, but .... I don't know. This doesn't make any sense to me, but I suggested deprecating only those I understood.